### PR TITLE
feat: support `open` option

### PR DESCRIPTION
**Pull request by nonzzz**
### Description

This pull request is a feature support.

### Linked Issues

#69

### CODE REVIEW

File: src/node/index.ts [hunk 1]
```diff
@@ -423,29 +433,32 @@ export default function PluginInspect(options: Options = {}): Plugin {
...
+          console.warn('Parse resolved url failed:', error)
         }
+      }
 
+      if (!silent) {
         _print()
         // eslint-disable-next-line no-console
         console.log(`  ${c.green('➜')}  ${c.bold('Inspect')}: ${colorUrl(`${host}${base}__inspect/`)}`)
       }
+      // I can't get the config when the service starts.
```
Comment (antfu) [inline src/node/index.ts:459]: What do you mean by this line?

Comment (nonzzz) [reply src/node/index.ts:459]: I’ve try implmenet the logic outside of `printUrls`. But vite cann't get `config` during startup.So I can't get the correct route.Should i put the comment inside `printUrls` right?

File: src/node/index.ts [hunk 2]
```diff
@@ -517,6 +530,30 @@ export default function PluginInspect(options: Options = {}): Plugin {
     return targetDir
   }
 
+  function createInternalServer(staticPath: string) {
+    const server = createServer()
```
Comment (antfu) [inline src/node/index.ts:534]: Why do we need to create a server here?

Comment (nonzzz) [reply src/node/index.ts:534]: I want to create a static server hosting `.vite-inspect`

**Comment by antfu** [comment]
Thanks!

**Pull request merged**